3-extra/card-validator.js

```diff
@@ -0,0 +1,32 @@
+//creating card validator function
+
+function creditCardNumberValidator(cardNumber) {
+  let cardNumberElement = cardNumber.toString().split("");
+  if (!cardNumberElement.every((element) => /[0-9]/.test(element))) {
+    return false + "Your card is invalid!";
+  }
+  if (cardNumberElement.length !== 16) {
+    return false + ", Your card is invalid!";
+    //checks if card number has 16 digits.
+  }
+  if (cardNumberElement.every((element) => element !== element[0])) {
+    return false + ", Your card is invalid!";
+  }
+  if (cardNumberElement[cardNumberElement.length - 1] % 2) {
```

Comment:
This works fine; it's maybe a little more readable to use an expression that gives a boolean result eg `... % 2 !== 0`